apidoc/Titanium/Media/Media.yml

```diff
@@ -1526,7 +1526,9 @@ properties:
     default: false
   - name: mediaTypes 
     summary: | 
-        Array of media type constants to allow.
+        Array of media type constants to allow. If you want to allow live photos with
+        Titanium.Media.MEDIA_TYPE_LIVEPHOTO on iOS 9.1 and later, you also need to specify
+        at least Titanium.Media.MEDIA_TYPE_PHOTO as a fallback.
```

Comment:
Incorrect, because the camera doesn't support live photos (yet). That's why I didn't add it there :-)